### ISSUE TITLE
logging doesn't work in mvc

### ISSUE THREAD

**Issue by luntc1972** [stemarie/log4net.Azure#20]
I'm trying to use azure table logging an MVC app and the logs aren't being sent to azure.  I have a textappender and consoleappender that work fine.

**Comment by minhn** [comment]
ditto the above comment, works in a console app but not in an ASP.NET app

**Comment by minhn** [comment]
looks like the "issue" is the bufferSize, by default the BufferingAppenderSkeleton has a buffer size of 512, so in an ASP.NET application, the buffer isn't being written to Azure table or blob until the 513th log event.  This can be solved by changing the buffer size in the config <appender ...><bufferSize value="1"/></appender>.

**Comment by stemarie** [comment]
@luntc1972 If you are testing then feel free to call the Flush method, otherwise as minhn as suggested I would change the buffer size to allow for smaller value. I designed the appender to be as performant as possible this is why I used the BufferingAppenderSkeleton base class for both AzureBlobAppender and AzureTableAppender and didn't alter the default behavior.

Of course a good practice may also to call the Flush method when the application unloads so that there is no loss of logs.
